src/components/Layout/NestedScrollLayoutContext.tsx

```diff
@@ -0,0 +1,41 @@
+import { createContext, PropsWithChildren, useEffect, useMemo, useRef } from "react";
+import { Css, Only, Xss } from "src/Css";
+
+interface NestedScrollLayoutContextProps {
+  scrollableEl: HTMLElement;
+}
+
+export const NestedScrollLayoutContext = createContext<NestedScrollLayoutContextProps>({
+  scrollableEl: document.createElement("div"),
+});
+
+// Allow any css to be applied to the NestedScrollLayout container.
+type NestedScrollXss = Xss<any>;
+interface NestedScrollLayoutContextProviderProps<X> {
+  xss?: X;
```

Comment:
Since you're not putting any limits on this (which I'm good with, makes sense for this), you could do just `xss?: Properties` w/ the `Properties` from the `Css.ts` file.

And skip the `type ...` on line 13`, the `Only` on line 18, etc...